src/sources/host_metrics/cgroups.rs

```diff
@@ -364,15 +442,11 @@ define_stat_struct! { MemoryStat(
 )}
 
 fn is_dir(path: impl AsRef<Path>) -> bool {
-    std::fs::metadata(path.as_ref())
-        .map(|metadata| metadata.is_dir())
-        .unwrap_or(false)
+    std::fs::metadata(path.as_ref()).map_or(false, |metadata| metadata.is_dir())
```

Comment:
Any particular reason we're not `use`ing here, rather than qualifying?

Reply:
Pretty much that it's just used once AFAICR.